br/Makefile

```diff
@@ -31,6 +31,7 @@ TEST_PARALLEL  ?= 8
 PD_ADDR        ?= 127.0.0.1:2379
 BR_LOCAL_STORE ?= /tmp/backup_restore_test
 API_VERSION    ?= 1
+CLUSTER_VERSION?= nightly
```

Comment:
Nit: add a space before `?=` and align other lines.

Reply:
Done